src/test/java/com/google/api/codegen/testdata/ruby_smoke_test_library.baseline

```diff
@@ -0,0 +1,41 @@
+============== file: test/library/v1/library_service_smoke_test.rb ==============
+# Copyright 2017, Google Inc. All rights reserved.#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# AUTO GENERATED BY GAPIC
+
+require "minitest/autorun"
+require "minitest/spec"
+require "library/v1/library_service_client"
+
+describe "LibraryServiceSmokeTest" do
+  it "runs one smoke test with update_book" do
+    if ARGV.empty?
+      fail "Usage: #{$0} <project_id>"
+    end
+    project_id = ARGV[0].freeze
```

Comment:
I am thinking this would be better as an environment variable rather than an argument.
This will allow Rake to be able to run the test as well as being able to run the test from cli. This is because rake runs tests by require-ing the file and there is no way of passing an argument to a file using require. 

This part of code will end up looking like this if changed:
```rb
unless ENV["SMOKE_TEST_PROJECT"]
  fail "Usage: SMOKE_TEST_PROJECT=<project_id> ruby library_service_smoke_test.rb"
end
project_id = ENV["SMOKE_TEST_PROJECT"].freeze
```

For reference here is the bit in the Rakefile that I have in mind.
```rb
# Smoke tests
desc "Runs the smoke tests."
task :smoke_test, :project do |t, args|
  project = args[:project]
  project ||= ENV["SMOKE_TEST_PROJECT"]
  if project.nil?
    fail "You must provide a project. e.g. rake smoke_test[test123] or SMOKE_TEST_PROJECT=test123 rake smoke_test"
  end

  $LOAD_PATH.unshift "lib", "smoke_test"
  Dir.glob("smoke_test/**/*smoke_test.rb").each { |file| require_relative file }
end